src/controllers/QueryRunner.ts

```diff
@@ -295,7 +295,7 @@ export default class QueryRunner {
                         for (let row of result.resultSubset.rows) {
                             // iterate over the cells we want from that row
                             for (let cell = range.fromCell; cell <= range.toCell; cell++) {
-                                if (self.shouldRemoveNewLines() && row[cell]) {
+                                if (row[cell] && self.shouldRemoveNewLines()) {
```

Comment:
I am not sure. This works currently, but JavaScript as some subtleties to it that might matter for things like this.